tests/recursive_logging.rs

```diff
@@ -29,7 +29,7 @@ fn test_meta_logging_in_format_works() {
         .max_file_size(1024 * 1024)
         .build("logs")
         .unwrap();
-    let (writer, _guard) = NonBlockingBuilder::default().finish(rolling);
+    let (writer, _guard) = rolling_file::non_blocking_builder().finish(rolling);
```

Comment:
maybe `rolling_file::non_blocking().finish(rolling)` or even `rolling_file::non_blocking(rolling).finish()`

Reply:
`rolling_file::non_blocking(rolling).finish()` looks good. Let me give it a try.

Reply:
See https://github.com/fast/logforth/pull/73